plugins/PrivacyManager/API.php

```diff
@@ -218,14 +218,15 @@ public function setDeleteReportsSettings($enableDeleteReports = 0, $deleteReport
 
         $settings['delete_reports_older_than'] = $deleteReportsOlderThan;
 
-        $settings['delete_reports_keep_basic_metrics']   = (int) $keepBasic;
-        $settings['delete_reports_keep_day_reports']     = (int) $keepDay;
-        $settings['delete_reports_keep_week_reports']    = (int) $keepWeek;
-        $settings['delete_reports_keep_month_reports']   = (int) $keepMonth;
-        $settings['delete_reports_keep_year_reports']    = (int) $keepYear;
-        $settings['delete_reports_keep_range_reports']   = (int) $keepRange;
-        $settings['delete_reports_keep_segment_reports'] = (int) $keepSegments;
-        $settings['delete_logs_max_rows_per_query']      = PiwikConfig::getInstance()->Deletelogs['delete_logs_max_rows_per_query'];
+        $settings['delete_reports_keep_basic_metrics']             = (int) $keepBasic;
+        $settings['delete_reports_keep_day_reports']               = (int) $keepDay;
+        $settings['delete_reports_keep_week_reports']              = (int) $keepWeek;
+        $settings['delete_reports_keep_month_reports']             = (int) $keepMonth;
+        $settings['delete_reports_keep_year_reports']              = (int) $keepYear;
+        $settings['delete_reports_keep_range_reports']             = (int) $keepRange;
+        $settings['delete_reports_keep_segment_reports']           = (int) $keepSegments;
+        $settings['delete_logs_max_rows_per_query']                = PiwikConfig::getInstance()->Deletelogs['delete_logs_max_rows_per_query'];
+        $settings['delete_logs_unused_actions_max_rows_per_query'] = PiwikConfig::getInstance()->Deletelogs['delete_logs_unused_actions_max_rows_per_query'];
```

Comment:
AFAIK it shouldn't be needed to add this here?

Reply:
I'll remove it then :) thanks

Reply:
Given that we have the new setting present in Controller.php, I assume we need it here as well.